pkg/mco/machineconfig_test.go

```diff
@@ -9,6 +9,10 @@ import (
 
 const defaultMachineConfigName = "test-machine-config"
 
+var testSchemes = []clients.SchemeAttacher{
+	mcv1.Install,
```

Comment:
Install or addToSchem?

Reply:
@kononovn The [api docs](https://pkg.go.dev/github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1#pkg-variables) say that `AddToScheme` is deprecated.

```go
// AddToScheme is DEPRECATED
AddToScheme = Install
```